### PR TITLE
Bug 1501007 - terminate the queueservice so expirequeues exits

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -590,6 +590,7 @@ let load = loader({
 
       monitor.count('expire-queues.done');
       monitor.stopResourceMonitoring();
+      queueservice.terminate();
```
Comment (owlishDeveloper) [inline src/main.js]: `queueService`?

Comment (djmitche) [reply src/main.js]: It's a component that interfaces with Azure queues.
https://github.com/taskcluster/taskcluster-queue/blob/master/src/queueservice.js#L72

It has a cache, and a `setInterval` to clear that cache every 25 hours.  That interval timer causes node to continue running forever.  Calling `terminate` cancels the interval.

Comment (owlishDeveloper) [reply src/main.js]: I meant: isn't that a typo? Need capital `S`

Comment (djmitche) [reply src/main.js]: Oh, haha, I misunderstood the question.  Yes, precisely.  I apparently failed to copy this change out of the pod where I was experimenting, too.

Comment (owlishDeveloper) [reply src/main.js]: (thank you for the explanation tho! I didn't realize that)

**Comment by imbstack** [comment]
Tests are still failing but these look more like flakes?

**Comment by owlishDeveloper** [comment]
I wonder if the second failure is actually related to this change?

**Comment by djmitche** [comment]
They look flaky, yeah.

**Pull request merged**